### ISSUE TITLE
WASI executable

### ISSUE THREAD

**Issue by rcarmo** [Gozala/wisp#175]
I’ve been pondering how to get wisp running under a-shell (https://github.com/holzschu/a-shell) as a WASI CLI  executable. Is there any prior art?

**Comment by chr15m** [comment]
On the a-shell README it mentions there is a js runtime availble. Could you just run the Wisp output directly with that?

BTW I highly recommend checking out the [squint-cljs](https://github.com/squint-cljs/squint) suite as it has largely similar goals to Wisp but is more up to date and closer to Clojure.

**Comment by rcarmo** [comment]
Nope, `jsc` doesn’t really do the same as `node`, and `wisp.js` is not structured correctly to handle command line arguments, so it’s kind of useless as a CLI.
